### DISCUSSION TITLE
automation issues

### DISCUSSION THREAD

**Discussion by sodasam2243** [alandtse/tesla#874] [General]
Hello Home Assistant Tesla friends,

Got Issues with my automations.
non of my automations works.
idk if im doing it wrong or if there is an different issue.

1st automation setting temperature for my tesla after leaving work.
i set a geofence when leaving my work place. so my car can check the temp. and after my shower it can set the temperature.
![IMG_2333](https://github.com/alandtse/tesla/assets/104204543/5779f53b-8d53-4110-885e-a53deaa02622)
![IMG_2334](https://github.com/alandtse/tesla/assets/104204543/2ac01409-c8ba-4974-b804-64e2f9b4f7c5)

2nd opening frunk, when leaving work and arriving at my car.
leaving my work place and after arriving at my car opening my frunk
![IMG_2335](https://github.com/alandtse/tesla/assets/104204543/6b315e2a-c48a-4f35-888e-ff054f069c69)

3rd opening frunk, after left work and arrived at home.
only when im leaving working place and arriving after that at home opening frunk.
![IMG_2336](https://github.com/alandtse/tesla/assets/104204543/af28b2f1-cfe6-4577-92f4-7905e17c0ea7)

did I have done something wrong or is that just not working?

kindly regards

**Comment by skipishere** [comment]
Could you include the logs as per the template and include information such as what version of the integration, car version and model. Without any of that information we really have no idea if it's an issue with the service, your automation or something else.

If you can't perform any commands at all you might be hitting the API issue that is pinned (#774)

**Reply by sodasam2243** [reply]
thanks for your fast reply

my bad. home assistant is running on 2024.2.4
tesla integegration is running on latest software
it is a model y long rang 2022. i can control my car via the integration with no issues.
API shouldnt be the issue. even I dont get any errors.

i really dont know now how to do the the logging thing. i watched couple of videos. still dont get it. im pretty new in this home assistant thing

**Reply by skipishere** [reply]
Ok that helps that command are working.

Your first automation is checking if the temperature is above 24 and below 18 as a condition, this logic will never be true as both parts must pass (eg if the temperature is 30 in the car it is above 24 but it isn't below 18)

https://www.home-assistant.io/docs/scripts/conditions/#numeric-state-condition

**Reply by skipishere** [reply]
The next two automations also have a problem with the condition, conditions need to be `true` at the point the trigger activated.

In the last example you trigger when your phone leaves work but the condition is that the car is at home. The only time this would work would be if you left the car at home and then your phone left work (which wouldn't be ideal).

I'd probably recommend against having an automation that opens your frunk based on location as there is a high chance of it accidently activating when you don't mean for iit (eg you leave work to get some food but don't use your car)

**Reply by sodasam2243** [reply]
thank you very much for your effort. except the third one i was able to solve the other ones.

I was also concerned about that. When I thought about that. I only use my frunk for my work backpack.
our working place and parking spot is huge. I set the conditions there shouldn't be any issues with that.
I still activated notification if there will be any issues with accidentally opening. So I can remove the automation.

I will give it still a shot. Im actually on vacation. curious if this will work when I’m back.

Again thank you very much for all your efforts. :)

**Comment by alandtse** [comment]
This looks more like automation help and isn't related to the underlying component. It should be a discussion then.
